guides/common/modules/proc_viewing-installable-errata.adoc

```diff
@@ -0,0 +1,13 @@
+[id="Viewing_installable_errata_{context}"]
+= Viewing Installable Errata
+
+Use the following procedure to view a list of installable errata and select errata to install.
```

Comment:
We should either shorten this sentence or reword the file name, anchor, and heading to something like "Apply Installable Errata".